src/modules/ProductLicense.rb

```diff
@@ -1144,12 +1144,24 @@ def AskLicenseAgreement(src_id, dir, patterns, action, enable_back, base_product
       end
 
       licenses_ref = arg_ref(licenses)
-      DisplayLicenseDialog(
+
+      label = Pkg::SourceGeneralData(src_id)["name"]
+      title = _("License Agreement")
+
+      if !label.empty?
+        title = _("%s License Agreement") % label
+      end
+
+      eula_url = Pkg::SourceGeneralData(src_id)["url"]
+      Yast::ProductLicense.license_file_print = eula_url
```

Comment:
The URL should point to the license file which can be downloaded, actually it is `repository_url` + (`license.zip` or `license.tar.gz`). The used file should be stored in `@license_file_print`, see https://github.com/yast/yast-packager/blob/master/src/modules/ProductLicense.rb#L651.

Moreover you should leave the default location for `cd://` or `dvd://` URLs as there is nothing to "download" in that case.